.changeset/tasty-nails-fry.md

```diff
@@ -0,0 +1,14 @@
+---
+"thirdweb": minor
+---
+
+Adds findUsers function to query users on the server
+
+```ts
+import { findUsers } from "thirdweb/wallets";
+
+const users = await findUsers({
```

Comment:
got confused by the name of the function.

how about `getUserDetails()` or `getUser()` especially since you can only get 1 user result

Reply:
The return type is an array (I'm not sure if that's just a quirk of the backend or you can actually get multiple)

Reply:
Yeah you can get multiple because I could have two accounts with the same email (e.g. discord and google)

Reply:
are you sure? im pretty sure if its the same email, it resolves to the same 'user' (same wallet address)

Reply:
You're right, it autolinks if the emails are the same (even if I used a different auth method). Updated the function name